packages/CopyInput/src/CopyInputInputPropsCollector.js

```diff
@@ -0,0 +1,6 @@
+// Shell component for enhancing DX (developer experience)
+export default function CopyInputInputPropsCollector() {
```

Comment:
this one should return `<></>` a fragment so ReatDocGen can pick it up and report it back in the ReadMe File otherwise DocGen will interpret the null as a regular function and not a React Component. 

After doing that you can go to `package.json` and add it as a subcomponent so the generateReadMe file can find it while visiting the package like here: https://github.com/acl-services/paprika/blob/master/packages/DatePicker/package.json#L37